pkg/sql/sequence.go

```diff
@@ -287,30 +287,7 @@ func (p *planner) SetSequenceValueByID(
 	if !descriptor.IsSequence() {
 		return sqlerrors.NewWrongObjectTypeError(seqName, "sequence")
 	}
-	if err := setSequenceValueHelper(ctx, p, descriptor, newVal, isCalled, seqName); err != nil {
-		return err
-	}
 
-	// Clear out the cache and update the last value if needed.
-	p.sessionDataMutatorIterator.applyOnEachMutator(func(m sessionDataMutator) {
-		m.initSequenceCache()
-		if isCalled {
-			m.RecordLatestSequenceVal(seqID, newVal)
-		}
-	})
-	return nil
-}
-
-// setSequenceValueHelper is shared by SetSequenceValue and SetSequenceValueByID
-// to set the given sequence to a new given value.
-func setSequenceValueHelper(
```

Comment:
This used to be called by both `SetSequenceValue` and `SetSequenceValueByID`, but `SetSequenceValue` no longer exists so it can be inlined inside `SetSequenceValueByID` to remove a layer of err checking and arg copying.